terraform/install.sh

```diff
@@ -75,22 +75,35 @@ getBillingAccount() {
 }
 
 getProject() {
-  log "Checking for project list"
+  log "Checking for project list..."
+  acct=$(gcloud info --format="value(config.account)")
+  # get projects with user account
+  owner_projects=$(gcloud projects list --filter="id:stackdriver-sandbox-* AND name='Stackdriver Sandbox Demo'" --format="value(projectId)")
+  # get projects with billing account
   billed_projects=$(gcloud beta billing projects list --billing-account="$billing_id" --filter="project_id:stackdriver-sandbox-*" --format="value(projectId)")
```

Comment:
Why do you need "owner_projects" and "billed_projects"?

Reply:
Because "gcloud projects list" won't tell you anything about the billing info. And not all billing project lets you get IAM if you don't have at least browser permission. So I have to combine them.

Reply:
Since we only care about projects with owner permission, do we need the ones without browser permission?

Reply:
I mean if you use "gcloud projects get-iam-policy" on projects that you don't have at least browser permission, it will throw an error. And there are many projects under the billing account that you may not have permission for the current user account.

Reply:
We can ignore projects that throw an error though, right? We know those won't make it onto the list we display to the user